packages/models/organizations.coffee

```diff
@@ -1,4 +1,11 @@
 Organizations = new Mongo.Collection('organizations')
+Organizations.allow
+  insert: (userId, doc)  ->
+    false
+  update: (userId, doc, fields, modifier) ->
+    false
+  remove: (userId, doc) ->
+    false
```

Comment:
Oh yeah sorry, I meant to remove that in the latest merge but accidentally left it